### PR TITLE
Change LanguageVersionFacts.ToDisplayString() to 7.0

### LINKED ISSUE #18991: LanguageVersionFacts.ToDisplayString() should return "7.0" instead of "7"
```C#
            CreateStandardCompilation(text, parseOptions: TestOptions.Regular7).VerifyDiagnostics(
                // (3,23): error CS8503: The modifier 'static' is not valid for this item in C# 7. Please use language version 7.1 or greater.
```

### PR DESCRIPTION
**Pull request by reaction1989**
This fixes #18991

### CODE REVIEW

**Comment by reaction1989** [comment]
@dotnet-bot retest

**Comment by reaction1989** [comment]
@dotnet-bot help

**Comment by reaction1989** [comment]
@dotnet-bot test this please

**Comment by reaction1989** [comment]
@dotnet-bot test this please

**Comment by gafter** [comment]
Please retarget to the 15.6 branch.

**Comment by reaction1989** [comment]
@gafter done

File: src/Compilers/CSharp/Portable/CSharpResources.resx [hunk 1]
```diff
@@ -1,17 +1,17 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <root>
-  <!--
-    Microsoft ResX Schema
-
+  <!-- 
+    Microsoft ResX Schema 
+
```
Comment (jcouv) [inline src/Compilers/CSharp/Portable/CSharpResources.resx]: Nit: Please undo those trailing spaces.

**Comment by jcouv** [comment]
@dotnet-bot retest windows_debug_unit32_prtest please

File: src/Compilers/CSharp/Portable/CSharpResources.resx (hunk 1 above)
Comment (reaction1989) [inline src/Compilers/CSharp/Portable/CSharpResources.resx]: I love VS for that xD

Copy&Paste from Notepad++

**Comment by jcouv** [comment]
@gafter @dotnet/roslyn-compiler Please do a second review of this community contribution to fix the printing of C# version 7 to be "7.0". This is targeting 15.6 branch. Thanks

**Comment by jcouv** [comment]
@reaction1989 Could you rebase to resolve conflict?

**Comment by jaredpar** [comment]
@reaction1989 that last push added ~170 commits to this PR.  I think you may have merged in / rebased against master instead of dev15.6.

**Comment by reaction1989** [comment]
@jaredpar strange. I rebased against dev15.6 This happened probably because my branch was created from the master not from dev15.6 

I will look into it and make a new pr

**Pull request closed without merging**